packages/tracer/tests/e2e/allFeatures.decorator.test.functionCode.ts

```diff
@@ -9,8 +9,8 @@ const serviceName = process.env.EXPECTED_SERVICE_NAME ?? 'MyFunctionWithStandard
 const customAnnotationKey = process.env.EXPECTED_CUSTOM_ANNOTATION_KEY ?? 'myAnnotation';
 const customAnnotationValue = process.env.EXPECTED_CUSTOM_ANNOTATION_VALUE ?? 'myValue';
 const customMetadataKey = process.env.EXPECTED_CUSTOM_METADATA_KEY ?? 'myMetadata';
-const customMetadataValue = JSON.parse(process.env.EXPECTED_CUSTOM_METADATA_VALUE) ?? { bar: 'baz' };
-const customResponseValue = JSON.parse(process.env.EXPECTED_CUSTOM_RESPONSE_VALUE) ?? { foo: 'bar' };
+const customMetadataValue = process.env.EXPECTED_CUSTOM_METADATA_VALUE ? JSON.parse(process.env.EXPECTED_CUSTOM_METADATA_VALUE) : { bar: 'baz' };
+const customResponseValue = process.env.EXPECTED_CUSTOM_RESPONSE_VALUE ? JSON.parse(process.env.EXPECTED_CUSTOM_RESPONSE_VALUE) : { foo: 'bar' };
```

Comment:
Parsing a non-JSON / `undefined` value throws an error rather than evaluate to false, so if the goal was to use the default value (after the `??`) when one is not explicitly provided in the env, then this should be correct.

<img width="553" alt="image" src="https://user-images.githubusercontent.com/7353869/194330981-e1376c14-97c5-4bfc-abb8-2688a261b917.png">

Happy to revert if you think it's not ok or I misunderstood the intent of the original expression.

Reply:
No. I just want to check that it's really a missing edge case.